src/main/java/com/cs56fitnessapp/models/Meal.java

```diff
@@ -1,29 +1,137 @@
 package com.cs56fitnessapp.models;
+import java.util.ArrayList;
+
 
 /**
- * @author Nerissa Hsieh
+ * @author Nerissa Hsieh & Jimwell Castillo
  * Date Created: 10/03/17
  * Date Modified: 10/03/17
  */
 
-public class Meal implements DietFacts {
-    @Override
-    public int getCaloriesIn() {
-        return 0;
-    }
+public class Meal implements DietFacts
+    {
+        @Override
+        public int getCaloriesIn() {
+            return 0;
+        }
 
-    @Override
-    public double getTotalFatIn() {
-        return 0;
-    }
+        @Override
+        public double getTotalFatIn() {
+            return 0;
+        }
+
+        @Override
+        public double getTotalCarbIn() {
+            return 0;
+        }
+
+        @Override
+        public double getProteinIn() {
+            return 0;
+        }
+
+
+        /** Edited Jimwell Castillo
+         * 10.18.17
+         * */
+
+        public class Meals
+            {
+
+                private String mealName;
```

Comment:
As per UML, `title` field of the `Meal` class is not String type, but of `MealTitle` type which is an enumeration.